README.md

```diff
@@ -100,7 +103,7 @@ Which one of the following concepts does this best illustrate?
 What is the `ui-sref` directive, and how is it used?
 
 ```text
-Your answer goes here...
+It's similar to a helper method 'link_to' in Rails, and it's just how Angular(with ui.router as dependency) handles links.
```

Comment:
:+1: but more specifically it's what we use to load different states in our Angular app.